app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = t "wc"
```

Comment:
Đặt key I18n cụ thể hơn e nhé. wc là chi?

Reply:
dạ, wc là "Welcome to the Sample App!"

Reply:
bạn ghi tên biến rõ hơn nhé, đọc vô t tưởng nhà xí (yaoming)